### PR TITLE
[connectors] Remove GCS connector update support

### LINKED ISSUE #918: Dataproc Cluster creation fails with connectors.sh init action
Hi,
I am using the below for dataproc cluster creations:

```
gcloud dataproc clusters create xxx \
  --project xxx \
  --region europe-west1 \
  --zone europe-west1-b \
  --image-version 2.0-debian10 \
  --subnet xxx \
  --bucket xxx \
  --scopes 'https://www.googleapis.com/auth/cloud-platform' \
  --master-machine-type n1-standard-32 \
  --master-boot-disk-size 500 \
  --num-workers 6 \
  --worker-machine-type n1-standard-8 \
  --worker-boot-disk-size 500 \
  --optional-components JUPYTER \
  --enable-component-gateway \
  --initialization-actions gs://goog-dataproc-initialization-actions-europe-west1/connectors/connectors.sh \
  --initialization-action-timeout=59m \
  --metadata gcs-connector-version=2.2.0,bigquery-connector-version=1.2.0,spark-bigquery-connector-version=0.20.0
```

It ends with the cluster status ERROR. Tried finding the issue but seems nothing I could. Please help in this regard.
And when init actions are removed, the cluster starts fine.

PFA logs
[kannan_dataproc-initialization-script-0_output.txt](https://github.com/GoogleCloudDataproc/initialization-actions/files/6906868/kannan_dataproc-initialization-script-0_output.txt)
[kannan_dataproc-startup-script_output.txt](https://github.com/GoogleCloudDataproc/initialization-actions/files/6906869/kannan_dataproc-startup-script_output.txt)

### PR DESCRIPTION
**Pull request by medb**
Fixes #918

### CODE REVIEW

File: connectors/README.md [hunk 1]
```diff
@@ -23,64 +27,58 @@ on a [Google Cloud Dataproc](https://cloud.google.com/dataproc) cluster.
...
+then only this connector will be updated.
 
 For example:
 
-*   if Google Cloud Storage connector 2.2.0 version is specified and neither
-    Hadoop BigQuery connector not Spark BigQuery connector versions are
-    specified, then only Google Cloud Storage connector will be updated to 2.2.0
-    version:
+*   if Spark BigQuery connector 0.21.1 version is specified and Hadoop BigQuery
+    connector version is specified, then only Spark BigQuery connector version
```
Comment (Arun-Krishna) [inline connectors/README.md:74]: Change to `Hadoop BigQuery connector version is not specified`

**Review by Arun-Krishna — APPROVED** [review]
LGTM, one comment in readme.

File: connectors/connectors.sh [hunk 2]
```diff
@@ -10,20 +10,15 @@ readonly VM_CONNECTORS_DATAPROC_DIR=/usr/local/share/google/dataproc/lib
```
Comment (Deependra-Patel) [inline connectors/connectors.sh:10]: Let's remove "the Cloud Storage" on line number 3.

**Review by Deependra-Patel — APPROVED** [review]

File: connectors/connectors.sh (hunk 2 above)
Comment (medb) [reply connectors/connectors.sh:10]: Done.

File: connectors/README.md (hunk 1 above)
Comment (medb) [reply connectors/README.md]: Done.

**Pull request merged**